modules/fts/pages/fts-analyzers-search-functions.adoc

```diff
@@ -0,0 +1,33 @@
+= Analyzers - Search Functions
+
+Search functions allow users to execute full text search requests within a N1QL query. In the context of N1QL queries, a full text search index can be described as one of the following :
+* a) xref:https://docs.couchbase.com/server/current/n1ql/n1ql-language-reference/covering-indexes.html[Covering index]
+* b) Non-covering index
+This characterization depends on the extent to which it could answer all aspects of the SELECT predicate and the WHERE clauses of a N1Q1 query.
```

Comment:
"This characterization" -> "What index to use" ...

Reply:
This characterization seems to be more precise.